plugins/brightness/x/x.cpp

```diff
@@ -9,6 +9,23 @@ X::X()
     this->screen = QGuiApplication::primaryScreen();
 }
 
+bool X::is_supported()
+{
+    if (this->screen != nullptr) {
+        // Check that we can execute xrandr
+        QProcess process(this);
+        process.start(QString("xrandr --version"));
+        process.waitForFinished();
+        return process.exitCode() == 0;
+    }
+    return false;
+}
+
+uint8_t X::get_priority()
+{
+    return 1;
```

Comment:
shouldnt this have a higher priority than mocked?

Reply:
Ah, yeah forget to mention this one. Wasn't sure if `x` was supposed to be higher priority than `mocked`, I'll bump it up.